src/utils.cpp

```diff
@@ -113,16 +113,16 @@ void show_and_explain(int size, List* list, int array[], double ll_time,
                       double arr_time)
 {
 
-    std::cout << "~~~~~~~~~~~~~~~~~~~~Results~~~~~~~~~~~~~~~~~~~~" << std::endl;
+    std::cout << "~~~~~~~~~~~~~~~~~~~~~~~~~~~Results~~~~~~~~~~~~~~~~~~~~~~~~~~~" << std::endl;
     std::cout << "Data size: " << size << std::endl;
     std::cout << "Time taken for Array: " << arr_time << " seconds"
               << std::endl;
     std::cout << "Time taken for Linked-List: " << ll_time << " seconds"
               << std::endl;
     std::cout << "The percent difference between each algorithm is "
-              << ((abs(ll_time - arr_time)) / ((ll_time + arr_time) / (2)))
+              << (((abs(ll_time - arr_time)) / ((ll_time + arr_time) / (2))) * (100))
```

Comment:
Getting a warning from my editor:

`using integer absolute value function 'abs' when argument is of floating point type`